SourceryFramework/Sources/Parsing/Utils/Time.swift

```diff
@@ -9,5 +9,5 @@ import Foundation
 
 /// Returns current timestamp interval
 public func currentTimestamp() -> TimeInterval {
-    return CFAbsoluteTimeGetCurrent()
+    return Date().timeIntervalSince1970
```

Comment:
could we use currentTimestamp() everywhere we used `Date().timeInterval..` to avoid repetition?

Reply:
What a miss, thank you for noticing! Done ✅